src/python/pants/core/goals/lint.py

```diff
@@ -637,7 +643,7 @@ def key(fs: FieldSet) -> str:
     return Lint(_get_error_code(all_batch_results))
 
 
-@rule(level=LogLevel.DEBUG)
+@rule(level=LogLevel.TRACE)
```

Comment:
Why is this necessary to set? It should default to TRACE

Reply:
Ohhh I misunderstood why we were seeing the message. `LintResult` is a `EngineAwareReturnType` whose `level` is `INFO`.

Hmmm....